kakao/src/main/kotlin/com/agoda/kakao/edit/TextInputLayoutAssertions.kt

```diff
@@ -34,6 +37,15 @@ interface TextInputLayoutAssertions : BaseAssertions {
         })
     }
 
+    fun hasHint(@StringRes resId: Int) {
+        hasHint(getResourceString(resId))
+    }
+
+    private fun getResourceString(@StringRes resId: Int): String {
+        val targetContext: Context = InstrumentationRegistry.getInstrumentation().targetContext
+        return targetContext.resources.getString(resId)
+    }
```

Comment:
@Unlimity I just don't want to create private function for this case. I think it can be reused in the future

Reply:
I also think this function may be potentially shared. Feel like it may be worth extracting it to utility class, but on the other hand doing this now would be YAGNI violation 🤔

Let's me know if I should extract it (and if Yes - let me know where it should be located)